### PR TITLE
SC software release notes 21

### PR DESCRIPTION
**Pull request by pcm32**
Initial automated commit for the software release only (rel 20) for the cell type wheel and metadata search.

### CODE REVIEW

File: release-notes/sc/_posts/2022-06-29-20.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Data statistics
+
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This is a software release only, no additional data has been included.
+
+#### New features
+
+- Now cell types, organism parts and conditions (among other metadata fields) can be searched as well, in addition to the previous gene search.
+- The cell types, organism parts and conditions search has its results displayed by a the new interactive cell type wheel and gene v/s cell type heatmap.
```
Comment (alfonsomunozpomer) [inline release-notes/sc/_posts/2022-06-29-20.md]: I’d reword this to “Cell types, organism parts and condition search have their results displayed in an interactive cell type wheel accompanied by a gene vs cell type heatmap.”

File: release-notes/sc/_posts/2022-06-29-20.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- Expression heatmaps show cell types as well as calculated clusters.
```
Comment (alfonsomunozpomer) [inline release-notes/sc/_posts/2022-06-29-20.md]: This hasn’t been included in the release.

File: release-notes/sc/_posts/2022-06-29-20.md (hunk 1 above)
Comment (pcm32) [reply release-notes/sc/_posts/2022-06-29-20.md]:
```suggestion
- Cell types, organism parts and condition search have their results displayed in an interactive cell type wheel accompanied by a gene vs cell type heatmap.
```

File: release-notes/sc/_posts/2022-06-29-20.md (hunk 2 above)
Comment (pcm32) [reply release-notes/sc/_posts/2022-06-29-20.md]:
```suggestion
```

**Comment by pmb59** [comment]
@alfonsomunozpomer  I've converted this pull request to a draft, as I believe this could be Software Release 21

**Comment by anilthanki** [comment]
This is very old and the features are already announced in other releases so closing it
